src/sourmash/sourmash_args.py

```diff
@@ -22,11 +22,44 @@
 from . import signature as sigmod
 from .picklist import SignaturePicklist, PickStyle
 from .manifest import CollectionManifest
+import argparse
 
 
 DEFAULT_LOAD_K = 31
 
 
+def check_scaled_bounds(arg):
+    actual_min_val = 0
```

Comment:
what is the purpose of defining these here, since they're not used anywhere else?

Reply:
So should I not define these functions here then? I thought I was supposed to write them here in `src/sourmash/sourmash_args.py` and call them in `src/sourmash/command_sketch.py` and `src/sourmash/cli/utils.py`.

Reply:
sorry, I meant the constants (`actual_min_val` etc.)

Reply:
Oh alright, I'll remove them then!